documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -446,6 +446,7 @@
          }
       ],
       "recovery_state":{
+         "time_since_last_recovered":1,
```

Comment:
It just occurred to me that we typically would use the units in the name, e.g. `seconds_since_last_recovered`

Reply:
Done